spec/models/form/lettings/questions/stock_owner_spec.rb

```diff
@@ -107,15 +107,17 @@
 
       let(:log) { create(:lettings_log) }
 
+      let(:non_stock_organisation) { create(:organisation, holds_own_stock: false) }
       let(:expected_opts) do
-        Organisation.all.each_with_object(options) do |organisation, hsh|
+        Organisation.where(holds_own_stock: true).each_with_object(options) do |organisation, hsh|
           hsh[organisation.id] = organisation.name
           hsh
         end
       end
 
-      it "shows all orgs" do
+      it "shows orgs where organisation holds own stock" do
         expect(question.displayed_answer_options(log, user)).to eq(expected_opts)
```

Comment:
small one, but now that you have limited the organisations here, have you put a binding in to check there is at least one?

Reply:
I didn't use binding, but I did change the to and not_to around, to see what I got, and yes there was at least one

Reply:
if there were no stock owning organisations then `expect({}).not_to eq({})` would fail

Reply:
And it did fail :)